src/components/Layout/Navbar.vue

```diff
@@ -11,9 +11,26 @@
         <v-btn id="nav-user" text to="/user">Account</v-btn>
         <v-btn id="nav-logout" text to="/logout">Logout</v-btn>
       </template>
-      <template v-if="!isLoggedIn">
-        <v-btn id="nav-login" text to="/login">Log in</v-btn>
-        <v-btn id="nav-create-account" ext color="primary" to="/create-account">Sign up</v-btn>
+      <v-menu bottom left v-if="($vuetify.breakpoint.width < 400)  && !isLoggedIn">
+        <template v-slot:activator="{ on, attrs }">
+          <v-btn class="dots-button" v-bind="attrs" v-on="on">
```

Comment:
Neat Picking: I added `.dot-button` to remove the `box-shadow` which was turned on by default. It also changed the `background-color` of the button to `transparent`, it was #f5f5f5 by default, which looked a bit off.